README.md

```diff
@@ -422,4 +422,45 @@ base_columns:
       concerns: has_next_version
 ```
 This example results into two columns, which will be added into every row of `users` table: 
-`cr_date: NOW()` and `version: 2`
\ No newline at end of file
+`cr_date: NOW()` and `version: 2`
+
+## Enable/disable primary key generation (ID column)
+We can describe rules for primary key generation in a special file `.conf.yml`:
+```yaml
+".conf.yml":
+  tables:
+    all_tables_do_not_have_generated_pk:
+      applies_to: "friends"
+      pk:
+        generate: false
+
+"friends.yml":
+  vlad:
+    name: Vlad
+    age: 29
+```
+where `friends` is a table name. The items of `applies_to:` could be a either a string or a regular expression. 
```

Comment:
We need to write that that
* regular expression starts with `/`
* applies_to can be a comma separated string like "table1, table2, /user.+"
* applies_to can be an array of strings, regexps or other arrays